src/editorApp/index.js

```diff
@@ -7,14 +7,11 @@ window.addEventListener( 'DOMContentLoaded', () => {
 	// eslint-disable-next-line camelcase
 	const SCANNABLE_POST_TYPE = edac_editor_app.active;
 
-	if ( SCANNABLE_POST_TYPE ) {
-		// eslint-disable-next-line camelcase
-		if ( edac_editor_app.authOk === '1' ) {
```

Comment:
This is also defined in the pro plugin localize calls as well and should be removed from there too.